google/ggrc-core
# GGRC-8198/8227 Remove 'Delete' icon while Evidence File/Evidence URL is loading

# Issue description

Unmap FILE\ULR button isn't hidden after click

# Steps to test the changes

1. Create an Audit
2. Attach any Evidence URL
3. Click on the 'Delete' icon while it's loading

_Actual result_: the Delete button is clickable and doesn't respond while the evidence is loading
_Expected_: 'Delete' icon should not be displayed while Evidence File/Evidence URL is loading

# Solution description

Hide URLS\FILES from UI before `destroy` request is finished

# Sanity checklist




# PR Review checklist
